cloud-storage-uploader/README.md

```diff
@@ -0,0 +1,194 @@
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# cloud-storage-uploader
+
+This action uploads files/folders to a [GCS][gcs] bucket. This is useful when
+you want upload build artifacts from your workflow.
+
+Paths to files that are successfully uploaded are set as output variables and can be
+used in subsequent steps.
+
+## Prerequisites
+
+- This action requires Google Cloud credentials that are authorized to upload
+  blobs to the specified bucket. See the Authorization section below for more
+  information.
+
+## Usage
+
+### For uploading a file
+
+```yaml
+steps:
+  - id: upload-file
+    uses: GoogleCloudPlatform/github-actions/cloud-storage-uploader@master
+    with:
+      path: /path/to/file
+      destination: bucket-name/file
+
+  # Example of using the output
+  - id: uploaded-files
+    uses: foo/bar@master
+    env:
+      file: ${{steps.upload-file.outputs.uploaded}}
+```
+
+The file will be uploaded to `gs://bucket-name/file`
+
+### For uploading a folder
+
+```yaml
+steps:
+  - id: upload-files
+    uses: GoogleCloudPlatform/github-actions/cloud-storage-uploader@master
+    with:
+      path: /path/to/folder
+      destination: bucket-name
+
+  # Example of using the output
+  - id: uploaded-files
+    uses: foo/bar@master
+    env:
+      files: ${{steps.upload-files.outputs.uploaded}}
+```
+
+If the folder has the following structure:
+
+```
+.
+└── folder
+    ├── file1
+    └── folder2
+        └── file2
+```
+
+The files will be uploaded to `gs://bucket-name/folder/file1`,`gs://bucket-name/folder/folder2/file2`
+
+## Inputs
+
+- `path`: (Required) The path to a file or folder that should be uploaded
+  to the bucket.
+
+  You can specify either the absolute path or the relative path from the action:
+
+  ```yaml
+  path: /path/to/file
+  ```
+
+  ```yaml
+  path: ../path/to/file
+  ```
+
+- `destination`: (Required) The destination for the file/folder in the form bucket-name or with
+  an optional prefix in the form bucket-name/prefix
+
+  ```yaml
+  destination: bucket-name
+  ```
+
+  In the above example, the file will be uploaded to gs://bucket-name/file
+
+  ```yaml
+  destination: bucket-name/prefix
+  ```
+
+  In the above example, the file will be uploaded to gs://bucket-name/prefix/file
+
+- `service_account_key`: (Optional) [Google Service Account JSON][sa] credentials as JSON or base64 encoded string,
+  typically sourced from a [GitHub Secret][gh-secret]. If unspecified, other
+  authentication methods are attempted.
+
+## Outputs
+
+List of successfully uploaded file(s).
+
+For example:
+
+```yaml
+steps:
+  - id: upload-file
+    uses: GoogleCloudPlatform/github-actions/cloud-storage-uploader@master
+    with:
+      path: /path/to/file
+      destination: bucket-name/file
+```
+
+will be available in future steps as the output "uploaded":
+
+```yaml
+- id: publish
+  uses: foo/bar@master
+  env:
+    file: ${{steps.upload-file.outputs.token}}
```

Comment:
I'm not sure I understand where `token` came from?

Reply:
oops, I meant to do outputs.uploaded